common-docs/windows-app.md

```diff
@@ -10,8 +10,8 @@ is an alternative to the web version for offline and portability purposes.
 
 #### Windows app deprecation
 
-MakeCode Windows app is being deprecated. Support for the Windows app
-will end after June 2022.
+MakeCode Windows app is deprecated. Support for the Windows app
```

Comment:
Please do a callout to the new PWA app. Otherwise it is confusing. We still have new app, for user it is one and the same.

Reply:
Do we have a docs page pointing at / describing that? I poked around for a second when updating this and didn't see one so I left it off

Reply:
Sounds good to remove base file